### PR TITLE
added support for specifying multiple files for flow playback (aka scenario support first dependencies) 

### PR DESCRIPTION
**Pull request by gato**
Hi Max, 

As we agreed in https://github.com/mitmproxy/mitmproxy/pull/432 i'm PR you with (server|client)_replay supporting a list of files and modified _readflow to read all of them. 
I've done it in DumpMaster (dump.py) but also in cmdline.py (for mitmproxy to support it as well)
I've also fixed a bug (that i previously added), replay_ignore_payload_params was not been passed to start_server_playback when called in mitmproxy (it was ok in mitmdump) 

Regards.
Marcelo

### CODE REVIEW

File: libmproxy/console/__init__.py [hunk 1]
```diff
@@ -496,11 +496,13 @@ def toggle_eventlog(self):
...
-    def _readflow(self, path):
-        path = os.path.expanduser(path)
+    def _readflow(self, paths):
         try:
-            f = file(path, "rb")
-            flows = list(flow.FlowReader(f).stream())
+            flows = []
+            for path in paths:
+                path = os.path.expanduser(path)
+                f = file(path, "rb")
```
Comment (mhils) [inline libmproxy/console/__init__.py]: Could you adjust this to `with open(path, "rb"):` here and below? We should make sure that we close all files.

**Comment by mhils** [comment]
Hey Marcelo,

sorry for the late reply - the last week was super hectic and this PR got buried below a bunch of other mails. Looks excellent for me so far, but I think we should fix the open file handle on the way as well. :-)

Cheers,
Max

**Comment by gato** [comment]
Hi Max, 
No problem.
I will change the open's behavior this afternoon and update the PR.
Regards.
Marcelo

**Comment by mhils** [comment]
:+1:

**Comment by gato** [comment]
done

**Comment by mhils** [comment]
Hi Marcelo,

I finally get around reviewing your PRs - thanks for the great work. I just merged this one (modulo your latest merge commit, which is why it is not shown as merged here). If you encounter any duplicate code as in this case, please don't refrain from refactoring and unifying functionality. :smiley: I cleaned this whole mess (which certainly has been there before your additions) up in https://github.com/mitmproxy/mitmproxy/commit/0ac3227b7bb27a76b6f49aa1605f1b1887a01a10 .

Cheers,
Max

**Comment by gato** [comment]
Hi max, 
I haven't found any issue 
Regards.
Marcelo

**Pull request closed without merging**